config.yaml

```diff
@@ -22,6 +26,8 @@ processors:
 exporters:
   awsxray:
   awsemf:
+    resource_to_telemetry_conversion:
+      enabled: true
```

Comment:
we can't enable it in the master default config. It will generate lots of unwanted dimensions for customers. Should it be applied to Prometheus Config file in your case?

Reply:
`ClusterName` is inserted as a resource label by `resourcedetectionprocessor`. If `resource_to_telemetry_conversion` is configured as `false`, metrics will not contain any labels injected by `resourcedetectionprocessor`, and that will cause `ClusterName` missing from each grouped metric as well. Reckoning that ClusterName is a key dimension used by ContainerInsight dashboard, I'm afraid this option needs to be set to `true` not only in the case of Prometheus but any case that `ClusterName` is injected in this way.

Reply:
Yes, in AWS OTel Collector docker image, we'll build-in a few configuration files to support different features [here](https://github.com/aws-observability/aws-otel-collector/tree/main/config). So in this config folder, we'll create one config for `Prometheus`(in your case) and another one for `eks/containerinsights`. All these feature related configuration files, we can enable `resourcedetectionprocessor` and `resource_to_telemetry_conversion` per our needs. But for the default config file(the one you modified), we'll leave the minimal configuration w/ only AWS xray and emf enabled.

Reply:
Done